spec/models/conversation_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_conversations_on_name  (name) UNIQUE
+#
+require 'rails_helper'
+
+RSpec.describe Conversation, type: :model do
+  subject(:conversation) { FactoryBot.build(:conversation) }
+
+  describe 'validation test' do
```

Comment:
I think it's not quite right
Usually describe is used for entities (who? what?)
For example: describe _'#user'_ , describe _'associations'_
You can read about it here: https://www.betterspecs.org/#describe

Reply:
conversation  is the essence. 
https://www.betterspecs.org/#describe -- I read, in general, according to the recommendations. But I'll take care of it )